src/hotspot/share/nmt/memReporter.cpp

```diff
@@ -272,6 +272,13 @@ void MemSummaryReporter::report_summary_of_type(MEMFLAGS flag,
 }
 
 void MemSummaryReporter::report_metadata(Metaspace::MetadataType type) const {
+
```

Comment:
You have an extra blank line here, but maybe that is intentional?

Reply:
Thanks Stefan! Newline was intentional, better spacing.